packages/core/data-transfer/lib/providers/local-file-source-provider.ts

```diff
@@ -104,10 +102,14 @@ class LocalFileSourceProvider implements ISourceProvider {
   }
 
   #getBackupStream() {
-    const { file, encryption, compression } = this.options;
+    const { encryption, compression } = this.options;
+
+    // This should be impossible as long as bootstrap was called first
+    if (!this.#filestream) {
+      throw new Error('Could not read file stream');
+    }
 
-    const fileStream = fs.createReadStream(file.path);
-    const streams: StreamItemArray = [fileStream];
+    const streams: StreamItemArray = [this.#filestream];
```

Comment:
Related to the comment above, but what happens if the stream is already closed after the first stage has finish?